### PR TITLE
[test] Fix NAMD valid prog. environments

### PR DESCRIPTION
**Pull request by ChristopherBignamini**
Closes [UES-1742](https://jira.cscs.ch/browse/UES-1742)

### CODE REVIEW

**Review by victorusu — APPROVED** [review]
`lgtm`

**Review by teojgo — APPROVED** [review]

File: cscs-checks/apps/namd/namd_check.py [hunk 1]
```diff
@@ -48,10 +48,8 @@ def adapt_valid_systems(self):
 
     @run_after('init')
     def adapt_valid_prog_environs(self):
-        if self.current_system.name == 'pilatus':
+        if self.current_system.name in ['eiger', 'pilatus']:
```
Comment (vkarak) [inline cscs-checks/apps/namd/namd_check.py:51]: I would generally use tuples for such checks.

File: cscs-checks/apps/namd/namd_check.py [hunk 2]
```diff
@@ -135,3 +127,10 @@ def setup_perf_vars(self):
                     'eiger:mc': {'days_ns': (0.05, None, 0.05, 'days/ns')},
                     'pilatus:mc': {'days_ns': (0.05, None, 0.05, 'days/ns')}
                 }
+
+    @performance_function('days/ns')
```
Comment (vkarak) [inline cscs-checks/apps/namd/namd_check.py:131]: In the future we should consider inverting this metric to `ns/day`, which is more standard.

**Comment by vkarak** [comment]
@jenkins-cscs retry daint dom

**Pull request merged**